Packs/GSuiteSecurityAlertCenter/Integrations/GSuiteSecurityAlertCenter/GSuiteSecurityAlertCenter.py

```diff
@@ -0,0 +1,719 @@
+                                             'Update Time'], removeNull=True)
+
+
+def prepare_hr_for_alert_feedback(feedbacks: List[Dict[str, Any]]) -> str:
+    """
+    Prepare the Human readable info for create alert feedback command.
+
+    :param feedbacks: The created feedback data.
+    :return: Human readable.
+    """
+
+    hr_table: List[Dict[str, Any]] = []
+    for feedback in feedbacks:
+        hr_table.append({
+            'Feedback ID': feedback.get('feedbackId', ''),
+            'Alert ID': feedback.get('alertId', ''),
+            'Create Time': feedback.get('createTime', ''),
+            'Feedback Type': feedback.get('type', ''),
+            'Email': feedback.get('email', '')
+        })
+    return tableToMarkdown('Feedback Details', hr_table,
+                           ['Feedback ID', 'Alert ID', 'Create Time', 'Feedback Type', 'Email'], removeNull=True)
+
+
+def prepare_hr_for_batch_command(response: Dict[str, Any], method: str) -> str:
+    """
+    Prepare the Human readable info for batch delete and undelete alerts command.
+
+    :param response: The delete and undelete alerts data.
+    :param method: To delete or undelete alerts.
+    :return: Human readable.
+    """
+
+    hr_list: List[Dict[str, Any]] = []
+    for each_success_id in response.get('successAlertIds', []):
+        hr_record = {
+            'Alert ID': each_success_id,
+            'Status': 'Success'
+        }
+        hr_list.append(hr_record)
+
+    for each_fail_key, val in response.get('failedAlertStatus', {}).items():
+        hr_record = {
+            'Alert ID': each_fail_key,
+            'Status': f'Fail ({val.get("message")})'
+        }
+        hr_list.append(hr_record)
+
+    return tableToMarkdown(
+        name=method,
+        t=hr_list,
+        headers=['Alert ID', 'Status'],
+        removeNull=True
+    )
+
+
+def create_custom_context_for_batch_command(response: Dict[str, Any]) -> Tuple[List, List]:
+    """
+    Prepare the custom Context Output for batch delete and undelete alerts command.
+
+    :param response: The batch delete and undelete alerts data.
+    :return: Success alerts list and failed alerts list
+    """
+    success_list: List = []
+    failed_list: List = []
+    for each_id in response.get('successAlertIds', []):
+        success_obj: Dict[str, Any] = {
+            'id': each_id,
+            'status': 'Success'
+        }
+        success_list.append(success_obj)
+
+    for failed_key, value in response.get('failedAlertStatus', {}).items():
+        failed_alert_id: Dict[str, Any] = {
+            'id': failed_key,
+            'status': 'Fail',
+            'code': value.get('code'),
+            'message': value.get('message', '')
+        }
+        failed_list.append(failed_alert_id)
+
+    return success_list, failed_list
+
+
+def check_required_arguments(required_arguments: List[str], args: Dict[str, Any]):
```

Comment:
We have implemented this method because when we pass only whitespace in required argument the Cortex XSOAR allows the value and does not show required argument not present. Once we fetch the command arguments in code we use `GSuiteClient.strip_dict()` method which strips the value of all arguments and removes empty arguments. 

So, to check if all the required arguments are present before hitting the API, arguments are passed through this method.